src/style.css

```
@@ -1 +1,72 @@
 /* Styles go here */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgb(0 0 0 / 5%);
+}
+
+.nav {
+  display: block;
+}
+
+.list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.list_item {
+  margin-left: 20px;
+}
+
+.list_item:first-child {
+  margin-left: 0;
+}
+
+.list-link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.list-link:hover {
+  color: #00acdc;
+}
```

```suggestion
.list-link:hover,
.is-active {
  color: #00acdc;
}
```
Можна об'єднати ці два селектора. Тут однакові стилі